crypto/bn/bn_rsa_fips186_4.c

```diff
@@ -0,0 +1,303 @@
+/*
+ * Copyright 2018 The OpenSSL Project Authors. All Rights Reserved.
+ * Copyright (c) 2018, Oracle and/or its affiliates.  All rights reserved.
+ *
+ * Licensed under the OpenSSL license (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+#include <stdio.h>
+#include <openssl/bn.h>
+#include "bn_lcl.h"
+#include "internal/bn_int.h"
+
+/*
+ * FIPS 186-4 Table B.1. "Min length of auxilary primes p1, p2, q1, q2".
+ */
+static int bn_rsa_fips186_4_aux_prime_min_size(int nbits)
+{
+    int min_bits;
+    switch (nbits) {
+#if 0
+    case 1024:
+        min_bits = 101;
+        break;
+#endif
```

Comment:
A query for the OMC, should the `#if 0` parts be included in the case, left as is (there but not compiled) or removed entirely?  The FIPS table still lists the constants for the small end but other parts of the standards deny their use.